src/load/azext_load/data_plane/utils/utils.py

```diff
@@ -473,6 +503,27 @@ def create_or_update_test_with_config(
         new_body["loadTestConfiguration"]["splitAllCSVs"] = yaml_test_body[
             "loadTestConfiguration"
         ]["splitAllCSVs"]
+
+    logger.debug("autostop_criteria: %s", autostop_criteria)
+    logger.debug("yaml_test_body: %s", yaml_test_body["autoStopCriteria"])
+
+    if autostop_criteria is not None:
+        new_body["autoStopCriteria"] = {
+            "autoStopDisabled": autostop_criteria.get("autoStopDisabled", False),
+            "errorRate": autostop_criteria.get("errorRate", Decimal(90.0)),
+            "errorRateTimeWindowInSeconds": autostop_criteria.get("errorRateTimeWindowInSeconds", 60),
+        }
+    elif yaml_test_body.get("autoStopCriteria") is not None:
+        new_body["autoStopCriteria"] = yaml_test_body["autoStopCriteria"]
```

Comment:
just confirming if we validate all the properties inside the autostop criteria ? what if some of the properties are not valid